fluxx.game.php

```diff
@@ -166,6 +168,8 @@ protected function setupNewGame($players, $options = [])
     self::setGameStateInitialValue("activeRichBonus", 0);
     self::setGameStateInitialValue("activeFirstPlayRandom", 0);
     self::setGameStateInitialValue("forcedCard", -1);
+    self::setGameStateInitialValue("playerTurnUsedPartyBonus", -1);
```

Comment:
Should be init at 0?

Reply:
Yes, you are right, that's more correct - I will fix it.